src/trio/_subprocess.py

```diff
@@ -329,13 +329,13 @@ async def _open_process(
     want.
 
     Args:
-      command (list or str): The command to run. Typically this is a
-          sequence of strings such as ``['ls', '-l', 'directory with spaces']``,
-          where the first element names the executable to invoke and the other
-          elements specify its arguments. With ``shell=True`` in the
-          ``**options``, or on Windows, ``command`` may alternatively
-          be a string, which will be parsed following platform-dependent
-          :ref:`quoting rules <subprocess-quoting>`.
+      command (str or bytes or Sequence[str] or Sequence[bytes]): The command to
```

Comment:
what's the upside of documenting the type in the docstring at all? It feels redundant

Reply:
I don't know, but if they are documented, they should be documented accurately.

Reply:
oh for sure, but maybe better to remove them (unless we wanna leave that for a different issue+pr to do that systematically). The other parameters aren't specified in this docstring even.

Reply:
Do what you feel is best, so long as consistency is maintained.